### ISSUE TITLE
Help me make a test plan (DRAFT)

### ISSUE THREAD

**Issue by lachlan-00** [ampache/ampache#1881]
# Test Plan

**This is a draft of what i will do when moving from branch to branch. i need a document i can run through to make sure the changes have not created any bugs.**

**i'm not a professional software dev so if anyone has developed a test plan before help would be appreciated!**

When moving from one branch to another a test must be performed on all areas affected by the pull.

Ideally; lib/class/* should not need extra testing when all features can be verified as functional.

### users.php

* Change
* Test
* Status

### license.php

* Change
* Test
* Status

### catalog.php

* Change
* Test
* Status

### modules.php

* Change
* Test
* Status

### mail.php

* Change
* Test
* Status

### system.php

* Change
* Test
* Status

### shout.php

* Change
* Test
* Status

### export.php

* Change
* Test
* Status

### duplicates.php

* Change
* Test
* Status

### access.php

* Change
* Test
* Status

### broadcast.inc

* Change
* Test
* Status

### calculate_art_size.inc

* Change
* Test
* Status

### catalog_update.inc

* Change
* Test
* Status

### channel_run.inc

* Change
* Test
* Status

### daap_register.bat

* Change
* Test
* Status

### delete_disabled.inc

* Change
* Test
* Status

### dump_album_art.inc

* Change
* Test
* Status

### fix_filenames.inc

* Change
* Test
* Status

### migrate_config.inc

* Change
* Test
* Status

### print_tags.inc

* Change
* Test
* Status

### sort_files.inc

* Change
* Test
* Status

### websocket_run.inc

* Change
* Test
* Status

### write_playlists.inc

* Change
* Test
* Status

## bin/install/*

* Change
* Test
* Status

## channel/*

* Change
* Test
* Status

## daap/*

* Change
* Test
* Status

## lib/*

* Change
* Test
* Status

## lib/class/*

* Change
* Test
* Status

### aurora.js

* Change
* Test
* Status

### Beets

* Change
* Test
* Status

### captcha

* Change
* Test
* Status

### catalog

* Change
* Test
* Status

### horde

* Change
* Test
* Status
### infotools

* Change
* Test
* Status

### jquery-jplayer-midnight-black

* Change
* Test
* Status

### jquery-mediaTable

* Change
* Test
* Status

### jquery-ui-ampache

* Change
* Test
* Status

### localplay

* Change
* Test
* Status

### oauth

* Change
* Test
* Status

### rhinoslider

* Change
* Test
* Status

### UberViz

* Change
* Test
* Status

### 7Digital

* Change
* Test
* Status

### Amazon

* Change
* Test
* Status

### Bitly

* Change
* Test
* Status

### CatalogFavorites

* Change
* Test
* Status

### ChartLyrics

* Change
* Test
* Status

### Discogs

* Change
* Test
* Status

### Facebook

* Change
* Test
* Status

### Flattr

* Change
* Test
* Status

### Flickr

* Change
* Test
* Status

### FriendsTimeline

* Change
* Test
* Status

### GoogleAnalytics

* Change
* Test
* Status

### GoogleMaps

* Change
* Test
* Status

### Gravatar

* Change
* Test
* Status

### Headphones

* Change
* Test
* Status
### Lastfm

* Change
* Test
* Status

### Libravatar

* Change
* Test
* Status

### Librefm

* Change
* Test
* Status

### LyricWiki

* Change
* Test
* Status

### Matomo

* Change
* Test
* Status

### MusicBrainz

* Change
* Test
* Status

### Omdb

* Change
* Test
* Status

### Paypal

* Change
* Test
* Status

### Piwik

* Change
* Test
* Status
### RSSView

* Change
* Test
* Status

### ShoutHome

* Change
* Test
* Status

### StreamBandwidth

* Change
* Test
* Status

### StreamHits

* Change
* Test
* Status

### StreamTime

* Change
* Test
* Status

### TheAudioDb

* Change
* Test
* Status

### Tmdb

* Change
* Test
* Status

### Tvdb

* Change
* Test
* Status

### Twitter

* Change
* Test
* Status

### Yourls

* Change
* Test
* Status

## play/*

* Change
* Test
* Status

### ajax.server.php

* Change
* Test
* Status

### browse.ajax.php

* Change
* Test
* Status

### catalog.ajax.php

* Change
* Test
* Status

### democratic.ajax.php

* Change
* Test
* Status

### edit.server.php

* Change
* Test
* Status

### fs.ajax.php

* Change
* Test
* Status

### index.ajax.php

* Change
* Test
* Status

### localplay.ajax.php

* Change
* Test
* Status

### player.ajax.php

* Change
* Test
* Status

### playlist.ajax.php

* Change
* Test
* Status

### podcast.ajax.php

* Change
* Test
* Status

### random.ajax.php

* Change
* Test
* Status

### refresh_reordered.server.php

* Change
* Test
* Status

### search.ajax.php

* Change
* Test
* Status
### song.ajax.php

* Change
* Test
* Status

### sse.server.php

* Change
* Test
* Status

### stats.ajax.php

* Change
* Test
* Status

### stream.ajax.php

* Change
* Test
* Status

### tag.ajax.php

* Change
* Test
* Status

### user.ajax.php

* Change
* Test
* Status

### xml.server.php

* Change
* Test
* Status

### cm-control-reply.php

* Change
* Test
* Status

### cm-event-reply.php

* Change
* Test
* Status

### control-reply.php

* Change
* Test
* Status

### event-reply.php

* Change
* Test
* Status

### find.php

* Change
* Test
* Status

### MediaServerServiceDesc.php

* Change
* Test
* Status

### playstatus.php

* Change
* Test
* Status

## webdav/*

* Change
* Test
* Status

## albums.php

* Change
* Test
* Status

## artists.php

* Change
* Test
* Status

## arts.php

* Change
* Test
* Status

## batch.php

* Change
* Test
* Status

## broadcast.php

* Change
* Test
* Status

## browse.php

* Change
* Test
* Status

## channel.php

* Change
* Test
* Status

## democratic.php

* Change
* Test
* Status

## graph.php

* Change
* Test
* Status

## image.php

* Change
* Test
* Status

## install.php

* Change
* Test
* Status

## labels.php

* Change
* Test
* Status

## localplay.php

* Change
* Test
* Status

## login.php

* Change
* Test
* Status

## logout.php

* Change
* Test
* Status

## lostpassword.php

* Change
* Test
* Status

## mashup.php

* Change
* Test
* Status

## phpinfo.php

* Change
* Test
* Status

## playlist.php

* Change
* Test
* Status

## podcast.php

* Change
* Test
* Status

## podcast_episode.php

* Change
* Test
* Status

## preferences.php

* Change
* Test
* Status

## mpvmsg.php

* Change
* Test
* Status

## radio.php

* Change
* Test
* Status

## random.php

* Change
* Test
* Status

## register.php

* Change
* Test
* Status

## rss.php

* Change
* Test
* Status

## search.php

* Change
* Test
* Status

## share.php

* Change
* Test
* Status

## shout.php

* Change
* Test
* Status

## show_get.php

* Change
* Test
* Status

## smartplaylist.php

* Change
* Test
* Status

## song.php

* Change
* Test
* Status

## stats.php

* Change
* Test
* Status

## stream.php

* Change
* Test
* Status

## test.php

* Change
* Test
* Status

## tvshow_seasons.php

* Change
* Test
* Status

## tvshows.php

* Change
* Test
* Status

## update.php

* Change
* Test
* Status

## upload.php

* Change
* Test
* Status

## util.php

* Change
* Test
* Status

## video.php

* Change
* Test
* Status

## waveform.php

* Change
* Test
* Status

## web_player.php

* Change
* Test
* Status

## web_player_embedded.php

* Change
* Test
* Status

**Comment by lachlan-00** [comment]
we're a long way from stable.
